### ISSUE TITLE
Issue generating colors for some palettes

### ISSUE THREAD

**Issue by chrislim5** [SymbolixAU/colourvalues#73]
When generating colors, some color palettes don't seem to space their values appropriately. For example:

```
test_vals <- runif(100, 0, 1)

colourvalues::color_values(
  test_vals, 
  palette = colourvalues::get_palette("spectral"), 
  n_summaries = 5,
  include_alpha = FALSE,
  na_colour = "#808080"
)
```
Gives summary values: 
`$summary_colours [1] "#9E0142" "#070707" "#000000" "#000000" "#000000"`

Inferno, viridis, matlab_like, blue2red, and sequential_hcl have worked fine for me. I have had problems with spectral and rdylbu (and similar palettes). I have not tested all of the palettes.

**Comment by dcooley** [comment]
Thanks for finding this - it's a bug in the `get_palette()` function. 

you can see this if you grab the palette directly

```r
test_vals <- runif(100, 0, 1)

colourvalues::color_values(
  test_vals, 
  palette = as.matrix( colourvalues:::spectral() ),
  n_summaries = 5,
  include_alpha = FALSE,
  na_colour = "#808080"
)

```

**Comment by dcooley** [comment]
@chrislim5 what version are you using - I think this was fixed in 0.3.7?

**Comment by chrislim5** [comment]
@dcooley I'm using colourvalues version 0.3.6 on R 3.6.1

**Comment by dcooley** [comment]
yeah this was fixed in the latest 0.3.7 update - as part of [this issue](https://github.com/SymbolixAU/colourvalues/issues/72). If you update from CRAN you should get the expected result.

**Comment by chrislim5** [comment]
@dcooley Thanks for the quick response!
